### ISSUE TITLE
Edge-based content: Found errors without a plausible explanation

### ISSUE THREAD

**Issue by francipvb** [nvaccess/nvda#10698]
### Steps to reproduce:

This occur in various well-known apps.

1. Open the start menu and write office.
1. Open the "Office" universal app.
1. Wait the app toload and then press <kbd>tab</kbd> key twice.

### Actual behavior:

Error beep is played. See this log chunk

```
ERROR - eventHandler.executeEvent (07:29:06.442) - MainThread (20396):
error executing event: gainFocus on <NVDAObjects.UIA.edge.EdgeNode object at 0x06F213D0> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 155, in executeEvent
  File "eventHandler.pyc", line 92, in __init__
  File "eventHandler.pyc", line 100, in next
  File "browseMode.pyc", line 1513, in event_gainFocus
  File "speech\__init__.pyc", line 1073, in speakTextInfo
  File "textInfos\__init__.pyc", line 529, in getControlFieldSpeech
  File "speech\__init__.pyc", line 1499, in getControlFieldSpeech
KeyError: 'nav'
```

### Expected behavior:

Should behave as normal.

#### NVDA installed/portable/running from source:

Installed and from source, both tested.

#### NVDA version:

Tested with alpha-19523,f25f33d3 and 2019.3 beta branch.

#### Windows version:

Windows 10 Pro x64 (version 1909)

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

Tried 2019.2.1 without errors.

**Comment by francipvb** [comment]
CC @josephsl @leonardder @michaelDCurran @Adriani90

**Comment by francipvb** [comment]
Updated description with missing information.

**Comment by francipvb** [comment]
Pressed the wrong button 😂

**Comment by leonardder** [comment]
I don't have the office universal app myself. Is there another app where this happens? It would help to have a list of those.

**Comment by francipvb** [comment]
Hello,


Yes, the issue happen with github web page under classic edge. When you
navigate to the new menu, either by tabbing or with virtual buffer
review. Another is the Twitter universal app.


Cheers,

-- 
Cuando tus fuerzas terminan, las de mi Dios comienzan.

**Comment by francipvb** [comment]
In youtube the issue is reproducible too when you land to the navigation section.

**Comment by Adriani90** [comment]
Closing as fixed in #10710.
